tests/func-tests/monitoring_test.go

```diff
@@ -143,13 +143,19 @@ func getAlertByName(alerts promApiv1.AlertsResult, alertName string) *promApiv1.
 }
 
 func verifyOperatorHealthMetricValue(promClient promApiv1.API, initialOperatorHealthMetricValue, alertImpact float64) {
-	Eventually(func() bool {
-		systemHealthMetricValue := getMetricValue(promClient, "kubevirt_hco_system_health_status")
-		operatorHealthMetricValue := getMetricValue(promClient, "kubevirt_hyperconverged_operator_health_status")
+	Eventually(func(g Gomega) {
+		if alertImpact >= initialOperatorHealthMetricValue {
+			systemHealthMetricValue := getMetricValue(promClient, "kubevirt_hco_system_health_status")
+			operatorHealthMetricValue := getMetricValue(promClient, "kubevirt_hyperconverged_operator_health_status")
+			expectedOperatorHealthMetricValue := math.Max(alertImpact, systemHealthMetricValue)
+
+			g.Expect(operatorHealthMetricValue).To(Equal(expectedOperatorHealthMetricValue),
+				"kubevirt_hyperconverged_operator_health_status value is %d, but its expected value is %d, "+
+					"while kubevirt_hco_system_health_status value is %d.",
```

Comment:
please replace `%d` with `%f` - the vars are float64, and in case of error, it will trigger a format error.

Reply:
done, thanks